libcudacxx/include/cuda/std/detail/libcxx/include/__verbose_abort

```diff
@@ -40,7 +40,7 @@ _CCCL_NORETURN _LIBCUDACXX_ATTRIBUTE_FORMAT(__printf__, 1, 2)
   _LIBCUDACXX_HIDE_FROM_ABI void __libcpp_verbose_abort(const char*, ...)
 {
   ::abort();
-  __builtin_unreachable(); // never reached, but needed to tell the compiler that the function never returns
+  _CCCL_UNREACHABLE(); // never reached, but needed to tell the compiler that the function never returns
```

Comment:
should this macro be called `_CCCL_BUILTIN_UNREACHABLE()`?

Reply:
It is unfortunately not a builtin but a pile of hacks that sometimes is a builtin